PowerPoint Unzipped File Experiments/OneImageAndPlainText - Readable XML Formatting.pptx/ppt/slideLayouts/slideLayout1.xml

```diff
@@ -177,9 +177,10 @@
           <a:bodyPr/>
           <a:lstStyle/>
           <a:p>
-            <a:fld id="{036EEA2E-C0C0-453E-845A-3EE4D9BD6F62}" type="datetimeFigureOut">
+            <a:fld id="{CD2AE7CE-E738-4E57-84D1-A5C283E0AC8B}" type="datetimeFigureOut">
```

Comment:
what are these UUIDs for?

Reply:
more importantly do we know how to keep them from changing as we build this lib. It'll add a lot of diff noise if it continues to update.

Reply:
Also in the experimental folder. But this ID seems to be generated one per pptx file. PptxgenJs actually had this ID totally hard-coded. Since we use the fragments, the ID we use won't change either.